README.md

```diff
@@ -22,13 +22,15 @@
 This is a [`settings`](https://github.com/asecurityteam/settings) that enables
 constructing a an event producer. The resulting client may be either an HTTP
```

Comment:
nit: the wording here should probably be tweaked away from "either" to "Currently, the only supported client is HTTP, as provided by ...".